src/setup-java.ts

```diff
@@ -16,6 +17,14 @@ async function run() {
 
     const matchersPath = path.join(__dirname, '..', '.github');
     console.log(`##[add-matcher]${path.join(matchersPath, 'java.json')}`);
+
+    const id = core.getInput('server-id', {required: false});
+    const username = core.getInput('username', {required: false});
+    const password = core.getInput('password', {required: false});
```

Comment:
Should we print a warning if a user sets one of these, but doesn't set them all?

Reply:
Will do

Reply:
I did this but then with the new env variable system I removed this as all inputs are totally optional now.